### ISSUE TITLE
File picker default highlight

### ISSUE THREAD

**Issue by mangas** [helix-editor/helix#3014]
#### Describe your feature request
I think it could be very handy to highlight the current file by default on the picker so that files from the same folder are close for selection.

**Comment by the-mikedavis** [comment]
To be clear, highlighting in this context is selecting the current file first and not syntax highlighting, correct?

So the behavior here would be that when the file picker's prompt is empty, the sort order should push the current file and files in its directory to the top?

**Comment by mangas** [comment]
That's correct, sorting the current file and directory up when no there is no filter

**Comment by pascalkuthe** [comment]
You mentinoed that #4666 was an alternative take on this issue. Does that mean that you view this feature-request as done now and this issue can be closed?

**Comment by mangas** [comment]
I would rather the approach of selecting the current file by default since it's more generic and one less keybinding. I seem to remember that @the-mikedavis was not a fan of this proposal so I leave this up to you guys. Personally I would rather work on and merge https://github.com/mangas/helix/tree/select-current-file-picker which
